src/install.rst

```diff
@@ -49,7 +49,7 @@ Install Nextstrain CLI
 
       .. note::
 
-         On newer Macs with an `Apple silicon chip <https://support.apple.com/en-us/HT211814>`_ (e.g. M1), `Rosetta 2 <https://support.apple.com/en-us/HT211861>`__ is required for both Nextstrain CLI itself and our runtimes.
+         On newer Macs with an `Apple silicon chip <https://support.apple.com/en-us/HT211814>`_ (e.g. M1), `Rosetta 2 <https://support.apple.com/en-us/HT211861>`__ is required for our runtimes.
```

Comment:
> it's (always?) been possible to build it from source on osx-arm64 without rosetta

Yes, indeed. That's because there's no binary bits in Nextstrain CLI itself. It's pure Python.

> Rosetta's not necessary for docker runtimes

Not necessary, but it will make the x86_64 code we still have in the arm64 image run faster (on [new enough Docker and macOS](https://www.docker.com/blog/docker-desktop-4-26/)). With Rosetta disabled, QEMU is used inside the Linux VM.

Reply:
[Discussion/decisions elsewhere](https://github.com/nextstrain/cli/pull/358#discussion_r1475134887) should inform the final form of the docs here.

Reply:
Resolved by https://github.com/nextstrain/cli/pull/361 and https://github.com/nextstrain/cli/pull/358.